docs/internal.md

```diff
@@ -59,6 +59,16 @@ Host agent:
 - `ha.stdout.log`: hostagent stdout (JSON lines, see `pkg/hostagent/events.Event`)
 - `ha.stderr.log`: hostagent stderr (human-readable messages)
 
+## Volume directory (`${LIMA_HOME}/_volumes/<VOLUME>`)
+
+A volume directory contains the following files:
+
+disk:
+- `datadisk`: the qcow2 disk that is attached to an instance when the volume is attached to an instance
+
+lock:
+- `in_use_by`: symlink to the instance directory that is using the volume
```

Comment:
relative symlink or absolute symlink?

Reply:
It is an absolute symlink, which I think is good, but not sure if it matters.

Reply:
I agree that an absolute symlink is preferable. This would let us further separate volume storage from instance storage if we wanted to (e.g. a separate `$LIMA_VOLUME_HOME`)

Reply:
> I agree that an absolute symlink is preferable.

It is already an absolute symlink iirc.

> This would let us further separate volume storage from instance storage if we wanted to (e.g. a separate $LIMA_VOLUME_HOME)

I can't really see the use case for this. If we wanted to do this at all, I would rather be able to specify a full pathname for the volume in `limactl volume create`, and then put a symlink to that file into `$LIMA_HOME/_volumes/$VOLUME`. That way you have total flexibility on where the volumes are stored, and you can also easily move them manually yourself.

Doing the manual moving should already work right now with this PR without further code changes.